### ISSUE TITLE
[Request] Saving the Nzb to Sections and Dateddir 

### ISSUE THREAD

**Issue by r0x0r316** [mbruel/ngPost#15]
For my example i will take the following monitor folders as given:
/mnt/linux
/mnt/freebsd
These are the Folders to be monitored 
The Base save path for nzbs will be
/mnt/nzbs 
The feature i search for is quiete easy to implement 
When a nzb is finished in monitor linux 
the nzb should be saved to 
/mnt/nzb/linux/2020-01-29/ubuntu.18.94.nzb
when a nzb is finished in freebsd it should be saved to
/mnt/nzb/freebsd/2020-01-29/freebsd.11.nzb
it would also be posstible to save it like this :
/mnt/nzbs/2020-01-29/freebsd/freebsd.11.nzb
/mnt/nzbs/2020-01-29/linux/ubuntu.18.04.nzb
---------------------------------------------------------------
so basically 
NZBBASEPATH/SECTION/DATE/NZBNAME
or
NZBBASEPATH/DATE/SECTION/NZBNAME

**Comment by mbruel** [comment]
This has been implemented but without the date folder, only one folder for each monitoring folder.
As not all people want this option, it's not active by default.
To have it, open [the config file](https://github.com/mbruel/ngPost/blob/master/ngPost.conf) and uncomment `MONITOR_NZB_FOLDERS = true`

I don't see any need to create a folder per date. you can just use ls or find to sort by date.
